src/figuro/widgets/input.nim

```diff
@@ -97,7 +98,7 @@ proc tick*(self: Input, now: MonoTime, delta: Duration) {.slot.} =
       self.cursorTick = (self.cursorTick + 1) mod 2
       refresh(self)
 
-proc clicked*(self: Input, kind: EventKind, buttons: set[UiMouse]) {.slot.} =
+proc activate*(self: Input, kind: EventKind = Done) =
```

Comment:
```suggestion
proc activate*(self: Input, kind: EventKind = Done) {.slot.} =
```

This would let other objects send a signal to activate the text input directly.

Reply:
Things like that could be handy in the future for automation and/or testing!